examples/matrix.py

```diff
@@ -24,15 +24,12 @@
 # Finally you can optionally specify a custom I2C address of the HT16k33 like:
 #matrix = matrix.Matrix16x8(i2c, address=0x70)
 
-# Clear the matrix.  Always call show after changing pixels to make the display
-# update visible!
+# Clear the matrix.
 matrix.fill(0)
-matrix.show()
```

Comment:
Maybe leave this in, but commented out, denoting that it is required if `auto_write == False`?

Reply:
I don't know if I agree with this. I can see where @sommersoft is coming from, but I think it can start bloating the simple examples to include all the potentially necessary code if you changed the default kwargs.

Reply:
I had something in there, but then took it out for same reason @kattni mentions. I think this is what I had, how about this:
```python
matrix.fill(0)
# If you turn auto_write off, you will need to call show() to update the display
# matrix.show()
```

Reply:
I can agree with the bloat concern. Having the parameters in the API documentation is enough for me.